CMakeLists.txt

```diff
@@ -37,21 +38,22 @@ if(is_big_endian)
 endif()
 
 cable_configure_compiler()
-add_compile_options(
+cable_add_compile_options(
     -Wcast-qual
     -Wcast-align
     -Wmissing-declarations
-    $<$<COMPILE_LANGUAGE:CXX>:-Wextra-semi>
-    $<$<COMPILE_LANGUAGE:CXX>:-Wold-style-cast>
+    IF_SUPPORTED
```

Comment:
So anything below the `IF_SUPPORTED` magic is only enabled if the compiler supports it?

How about `ONLY_IF_SUPPORTED`?

Reply:
Yes, these are optional and enabled if compiler of a particular language supports it.